src/components/slider/index.tsx

```diff
@@ -305,12 +305,12 @@ export default class Slider extends PureComponent<SliderProps, State> {
       const nonOverlappingTrackWidth = trackSize.width - this.initialThumbSize.width;
       const _x = Constants.isRTL && disableRTL ? nonOverlappingTrackWidth - x : x; // adjust for RTL
       this._thumbStyles.left = trackSize.width === 0 ? _x : (_x * nonOverlappingTrackWidth) / trackSize.width; // do not render above prefix\suffix icon\text
-      this.thumb.setNativeProps(this._thumbStyles);
+      this.thumb.current?.setNativeProps(this._thumbStyles);
     }
 
     if (this.minTrack) {
```

Comment:
You might need to check that `this.minTrack.current` is not undefined and not just `this.minTrack`